docs/plugins/kodiupdate.rst

```diff
@@ -16,6 +17,27 @@ which looks like this::
         user: kodi
         pwd: kodi
 
+With the basic configuration settings above, the ``kodiupdate`` plugin
+will notify Kodi to scan your entire music library after the import
+operation has completed. For large music libraries you may prefer to have
```

Comment:
It's not _quite_ the complete truth to say that the default configuration sends the request "after the import operation has completed"—it actually does this every time _anything_ changes in your library. For example, this happens after a `beet modify` where you update the metadata for a track or album.

Reply:
You're right, the default behavior is more general, but it's also more time consuming on a large library. For certain operations such as an import, it's much faster to process the few albums imported than re-scan the entire library. Maybe I should also look at getting notifications for beets modify? (Is there such a notification?)